.github/workflows/ci.yml

```
@@ -11,8 +11,20 @@ name: Flujo CI
 on:
   push:
     branches: [ "main" ]
+    tags: [ "v*" ]
   # Note: Pull requests are handled by pr-checks.yml for faster feedback
 
+# Global environment to keep installs deterministic and isolated per job
+env:
+  UV_VERSION: "0.4.24"
+  UV_LINK_MODE: copy
+  UV_CACHE_DIR: ${{ runner.temp }}/uvcache
```

actionlint (1.7.9)

context "runner" is not allowed here. available contexts are "github", "inputs", "secrets", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)